server/src/main/java/io/druid/server/security/AllowAllEscalator.java

```diff
@@ -21,23 +21,23 @@
 
 import com.metamx.http.client.HttpClient;
 
-/**
- * Singleton utility object that creates escalated HttpClients using a configuration-specified Authenticator's
- * getEscalatedClient() method.
- */
-public class AuthenticatorHttpClientWrapper
+public class AllowAllEscalator implements Escalator
```

Comment:
this is more of "NoopEscalator" , but I think it is named that way to match name with corresponding Authenticator impl used on the target server.

Reply:
renamed this to "NoopEscalator"